engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableSliceTest.java

```diff
@@ -433,6 +433,36 @@ private void doSliceTest(QueryTable table, String expected, int firstPositionInc
         assertEquals(expected, result);
     }
 
+    public void testSlicePct() {
+        final QueryTable table = TstUtils.testRefreshingTable(
+                RowSetFactory.fromRange(10, 19).toTracking(),
+                TableTools.charCol("letter", "9876543210".toCharArray()));
+
+        doSlicePctTest(table, "9876543210", 0, 1);
```

Comment:
Here you actually test the `slicePct(0,1)` case, and it does return the whole range.  This indicates that the docs need a rework to be more precise.